src/components/NoticeOfWithdrawal/RecordToBeWithdrawn.vue

```diff
@@ -102,5 +204,31 @@ export default class RecordToBeWithdrawn extends Mixins(DateMixin) {
     }
     return 'Invalid date'
   }
+
+  @Watch('partOfPoa')
+  private onpartOfPoaChange (newVal: boolean): void {
+    // If partOfPoa is false, reset hasTakenEffect to false
+    if (!newVal) {
+      this.hasTakenEffect = false
+    }
+  }
+
+  /** Emit plan of arrangement. */
+  @Watch('partOfPoa')
+  @Emit('partOfPoa')
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  emitPoa (partOfPoa: boolean): void {}
+
+  /** Emit come into effect. */
+  @Watch('hasTakenEffect')
+  @Emit('hasTakenEffect')
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  emitEffect (hasTakenEffect: boolean): void {}
 }
 </script>
+
+<style scoped>
+.checkbox-label {
+  font-size: 14px;
```

Comment:
We have a base class, `font-14`, to simplify cases like this :)

Reply:
Updated, thanks!